src/sentry/models/projectteam.py

```diff
@@ -16,23 +16,13 @@
 
 class ProjectTeamManager(BaseManager["ProjectTeam"]):
     def get_for_teams_with_org_cache(self, teams: Sequence["Team"]) -> QuerySet["ProjectTeam"]:
-        project_teams = (
+        return (
             self.filter(team__in=teams, project__status=ObjectStatus.ACTIVE)
             .order_by("project__name", "project__slug")
             .select_related("project")
+            .prefetch_related("project__organization")
```

Comment:
I think you can just add this to `select_related`, but up to you, both work. `prefetch_related` adds an extra query, `select_related` uses joins and so returns duplicate data per org

Reply:
yep, 100%. thanks for the suggestion.